### DISCUSSION TITLE
Allow setting page partial from a file using YAML format

### DISCUSSION THREAD

**Discussion by mmawhinney** [auth0/auth0-deploy-cli#1106] [Ideas]
### Checklist

- [X] I have looked into the [README](https://github.com/auth0/auth0-deploy-cli#readme) and have not found a suitable solution or answer.
- [X] I have looked into the [documentation](https://github.com/auth0/auth0-deploy-cli#documentation) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/auth0-deploy-cli/issues) and have not found a suitable solution or answer.
- [X] I have upgraded to the [latest version](https://github.com/auth0/auth0-deploy-cli/releases/latest) of this tool and the issue still persists.
- [X] I have searched the [Auth0 Community](https://community.auth0.com/tag/auth0-deploy-cli) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

I'd previously been managing a partial for my signup page using the Management API. I see that the CLI now supports managing partials. After exporting my current settings, I noticed that the raw template gets dumped in the `tenant.yml` file, eg:
```yaml
prompts:
  partials:
    signup:
      signup:
        form-content-end: >-
          <div>partial code...</div>
```

From what I've been able to find, the directory format supports referencing the template from a file, eg:
```json
"signup": [
  {
    "signup": [
      {
        "name": "form-content-end",
        "template": "./prompts/partials/signup/signup/form-content-end.html"
      }
    ]
  }
] 
```
But attempting to use a similar structure in the `tenant.yml` causes failures on import due to not being a valid schema.

### Expectation

It would be great to have support for referencing a template file in YAML as managing the template directly in the `tenant.yml` file is messy once you have more than a couple lines of HTML.

### Reproduction

1. Upload a partial using the Management API
2. Export tenant settings using the YAML format
3. Notice that the partial contents are exported directly into `tenant.yml` instead of a file

### Deploy CLI version

8.4.1

### Node version

18.19.0

**Comment by kushalshit27** [comment]
Hi, @mmawhinney 
Thanks for opening this issue. Removing the bug tag seems more like a feature support. 

Support referencing the template from a file similar to the directory format in YAML; it can be a breaking change for some users. This feature can be supported in the next major version releases. 

Thanks for your input for making this better!

**Comment by kushalshit27** [comment]
Hi, @mmawhinney 
We're moving this feature request to a GitHub discussion to encourage broader collaboration and input from the community.
